.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should be added to your git ignore. I'll help you manage this and make sure that the git cache is removed.